test/Elastic.Apm.AspNetFullFramework.Tests/Soap/SoapRequestTests.cs

```diff
@@ -26,12 +26,15 @@ public SoapRequestTests(ITestOutputHelper xUnitOutputHelper)
 		/// to parse the parameters for the web method.
 		/// </summary>
 		[AspNetFullFrameworkFact]

```

Comment:
```suggestion
		[AspNetFullFrameworkTheory]
```

Currently it fails:

```
System.InvalidOperationException : [Fact] methods are not allowed to have parameters. Did you mean to use [Theory]?
```

Due to the new parameter, it needs to be `AspNetFullFrameworkTheory`